src/runtime/test/fixtures/cmp-a.tsx

```diff
@@ -4,7 +4,6 @@ import { format } from './utils';
 
 @Component({
   tag: 'cmp-a',
-  styleUrl: 'cmp-a.css',
```

Comment:
The following change

https://github.com/ionic-team/stencil/pull/5090/files#diff-edc3612e034200b5e04705c4c0356e9bb8f1c7f5139df01a2f09995b1cee85edL114 

now triggers the compiler to actually include a `require` statement when compiling the component in the jest preprocessor. For some reasons Jest is not able to resolve the import. I tried to mock it without success. it doesn't seem like the `styleUrl` is important for any of the tests and I can also remove `src/runtime/test/fixtures/cmp-a.css` if we decide to move forward with this.

Reply:
This wasn't the right solution and got reverted